### DISCUSSION TITLE
Faikin 3d case

### DISCUSSION THREAD

**Discussion by naksu007** [revk/xESP32-Faikout#532] [Q&A]
Hi! I try order faikin case from JLC3DP. I use Faikin.stl file but JLC3DP says: Multiple parts are detected in your file, sorry to tell you that each file could only support 1 part for printing, please kindly split the multiple parts into single files to upload separately.

Is there two files which have one part of the case?

**Comment by revk** [comment]
You could separate them but not sure JLC can cope even if you do. Good luck. TY hey are designed for printing yourself really.

**Reply by naksu007** [reply]
Thank you for your reply. 

Can you tell me program which i can sepearate them?

I try two programs but i didn't succeed it.

**Comment by dahadd** [comment]
> Hi! I try order faikin case from JLC3DP. I use Faikin.stl file but JLC3DP says: Multiple parts are detected in your file, sorry to tell you that each file could only support 1 part for printing, please kindly split the multiple parts into single files to upload separately.
> 
> Is there two files which have one part of the case?

Hi 😉
Where you can find the file "faikin.stl" ? Thanks

**Reply by revk** [reply]
STL files are in the PCB directories.

I think most 3D print slicers will separate parts.
